### DISCUSSION TITLE
Welcome to textUtilis Discussions!

### DISCUSSION THREAD

**Discussion by GauriKashyap** [GauriKashyap/textUtilis#35] [Q&A] [answered]

**Comment by GauriKashyap** [comment]
How to make sure that the component used by the user in the application has some default value and is not blank? For example the Navbar component.

**Comment by hitaarthh** [comment, marked as answer]
This could be achieved by adding props to the given components. 

- Props stands for properties which a react components needs to have once it is called in the App.js file. These props can be changed for different component calls in order to use same component in different ways. For example a blog container. For different blogs the props of the container can be altered in a way to show different title, Blog description, images,redirection links and other blog details.
- For the above discussion, the prop should look like.

```JSX
Navbar.defaultProps = {
  title: "Set title here",
  aboutText: "About Text here"
}
```

**Reply by GauriKashyap** [reply]
Good Job, it was a very detailed explanation.
